scripts/sync-plugin.mjs

```diff
@@ -0,0 +1,34 @@
+import path from 'node:path';
+import {fileURLToPath} from 'node:url';
+
+import {fs} from '@appium/support';
```

Comment:
consider importing from `appium/support` instead

Reply:
you could also use the standard `fs/promises` instead of `support` wrappers

Reply:
it makes sense to use fs/promises